chrome/src/node/proxy-server.ts

```diff
@@ -0,0 +1,92 @@
+import WebSocket, * as ws from 'ws';
+import {
+  ProxyRequest,
+  UdpSendRequest,
+  UdpScanRequest,
+  UdpScanResponse,
+  UdpSendResponse,
+} from '../common/radio-message';
+import {NodeRadioController} from 'local-home-testing/build/src/radio';
+
+/**
+ * A WebSocket server that runs on node to provide radio fulfillment.
+ * Uses the `NodeRadioController` implementation to fulfill requests.
+ */
+export class ProxyRadioServer {
+  private webSocketServer: ws.Server;
+  private nodeRadioController: NodeRadioController;
+
+  constructor(port = 4321) {
```

Comment:
should that be configurable?

Reply:
Since there is no runtime interface to configure this default, I removed it as a parameter and added a global to make the `4321` port common.  There is the odd chance port `4321` is not available, so I opened issue #62.